### ISSUE TITLE
[Adaptive Cards> InputsWithRTL.json]: Placeholder text is not defined for 'Text input with an inline action with no icon' edit field.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#8975]
### Target Platforms

UWP

### SDK Version

1.0.1.0

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
Application: Adaptive Cards Universal
OS Build: 24H2 (OS Build 26252.5000)
App Version: 1.0.1.0

**Repro Steps:**
1. Open the Adaptive Cards Universal application.
2. Adaptive Card UWP Visualizer screen starts appearing.
3. Navigate to InputFormWithRTL.Json button by using Tab key and activate it.
4. InputFormRTL adaptive card will appear on right sight of the pane.
5. Navigate to InputFormwithRTL card by using Tab key.
6. Press the tab key and navigate to the edit field and observe the issue.

**Actual Result:**
For 'Name, Edit, Phone and Comments' edit field, there is no placeholder text is defined.

**Expected Result**
For 'Name, Edit, Phone and Comments' edit field, proper placeholder text should be defined. Such as 'Please enter your name' and so on.

**Note:**
Same issue repro throughout for all the edit fields.

**User Impact:**
People with cognitive disability will get impacted as user will not be able to know the purpose of the edit field or will get confused that what type of data needs to enter.

Have feedback to share on Bugs ? please tag bug as “A11yRCA” and add your feedback in the comment section.

### Screenshots

![There is no placeholder text is defined for the edit fields](https://github.com/user-attachments/assets/5501d247-bbcf-4395-bbba-a1aa45a9e8b0)

![No placeholder text is defined for the edit fields](https://github.com/user-attachments/assets/35bf79b9-b3f4-4590-8375-e8e8840f34d0)

### Card JSON

```JSON
NA
```
